docs/source/fortran_api.rst

```diff
@@ -2185,6 +2116,59 @@ LIBMUSCLE_Instance
     :r message: The received message.
     :rtype message: LIBMUSCLE_Message
 
+LIBMUSCLE_InstanceFlags
+```````````````````````
+
+.. f:type:: LIBMUSCLE_InstanceFlags
+
+    The InstanceFlags type represents the flags that can be supplied
+    when creating a new :f:type:`Instance`. Multiple flags may be set simultaneously,
+    for example:
+
+    .. code-block:: fortran
+
+        ports = LIBMUSCLE_PortsDescription_create()
+        ! Specify ports...
+        instance = LIBMUSCLE_Instance_create(ports, &
+            LIBMUSCLE_InstanceFlags(DONT_APPLY_OVERLAY=.true., USES_CHECKPOINT_API=.true.))
+        call LIBMUSCLE_PortsDescription_free(ports)
+
+
+    :f logical DONT_APPLY_OVERLAY: Set to ``.true.`` to not apply the received settings
+            overlay during prereceive of F_INIT messages.
+
+            If you're going to use Instance.receive_with_settings on your F_INIT ports,
+            you need to set this flag when creating an Instance.
+
+            If you don't know what that means, do not specify this flag and everything
+            will be fine. If it turns out that you did need to specify the flag, MUSCLE3
+            will tell you about it in an error message and you can add it still.
+
+    :f logical USES_CHECKPOINT_API: Set to ``.true.`` to indicate that this instance
+            supports checkpointing.
+
+            You may not use any checkpointing API calls when this flag is not supplied.
+
+    :f logical KEEPS_NO_STATE_FOR_NEXT_USE: Indicate this instance does not carry state
+            between iterations of the reuse loop.
+
+            This corresponds to :external:py:attr:`ymmsl.KeepsStateForNextUse.NO`.
+
+            If neither ``KEEPS_NO_STATE_FOR_NEXT_USE`` and
```

Comment:
"neither" ... "nor"

Also, I didn't get what "this corresponds to" means on first reading. Maybe turn it around, "to get the equivalent of KeepsStateForNextUse.NECESSARY, specify neither KEEPS_NO_STATE_FOR_NEXT_USE nor STATE_NOT_REQUIRED_FOR_NEXT_USE"? Or "By default, the instance is assumed to keep state between reuses, and to require that state (equivalent to ymmsl.KeepsStateForNextUse.NECESSARY)"?